manifest.konnector

```diff
@@ -8,7 +8,7 @@
   "slug": "sfr",
   "source": "git@github.com:konnectors/sfr.git",
   "editor": "Cozy",
-  "vendor_link": "https://www.sfr.fr/",
+  "vendor_link": "about:blank",
```

Comment:
Why is this necessary ?

Reply:
Très bonne question, ce n'est plus utile depuis que l'app amirale met le pilote sur about:blank automatiquement et je l'avais oublié. Je retire ça et je le retire sur red aussi. Merci :+1: